client_test.go

```diff
@@ -80,6 +80,33 @@ func TestClient_List(t *testing.T) {
 	})
 }
 
+func TestClient_Claim(t *testing.T) {
+	ctx := context.Background()
+	_, task2 := seedPair(ctx, t)
+
+	// shift task #2
+	claim1, err := client.Claim(ctx, task2.ID)
+	if err != nil {
+		t.Fatalf("expected no error, got %v", err)
+	}
+	defer claim1.Release(ctx)
+
+	// check claim
+	if exp, got := task2.ID, claim1.ID; exp != got {
+		t.Errorf("expected %v, got %v", exp, got)
+	}
+
+	// try again
+	if _, err := client.Claim(ctx, task2.ID); !errors.Is(err, ErrNoTask) {
```

Comment:
@mxmCherry there you go, if you try to claim an already claimed task, you get `ErrNoTask`